Methylseq/methylseq.sh

```diff
@@ -38,105 +43,160 @@ rsync -vur "$data_path/fastq/" $temp_path
-unmapped_R1_bismark_bam="${unmapped_R1_file_path}_bismark_bt2_pe.bam"
 
-unmapped_unmapped_R1="${temp_path}/$unmethyl_control/$hydroxymethyl_control/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz"
-unmapped_unmapped_R2="${temp_path}/$unmethyl_control/$hydroxymethyl_control/${trimmed_R2_file_name}.fastq.gz_unmapped_reads_2.fq.gz_unmapped_reads_2.fq.gz"
+#unmapped_R1_file_path=$(echo $R1_unmapped| sed 's/.fq.gz//')
+unmethyl_R1_file_path="${temp_path}/$unmethyl_control/$hydroxymethyl_control/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1"
+unmethyl_R1_file_name="${unmethyl_R1_file_path##*/}"
+R1_unmethyl_bismark_PE_report="${unmethyl_R1_file_name}_bismark_bt2_PE_report.txt"
+
+unmethyl_R1_bismark_bam="${unmethyl_R1_file_path}_bismark_bt2_pe.bam"
+
+hydroxy_R1_file_path="${temp_path}/$unmethyl_control/$hydroxymethyl_control/$methyl_control/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1"
+hydroxy_R1_file_name="${hydroxy_R1_file_path##*/}"
+R1_hydroxy_bismark_PE_report2="${hydroxy_R1_file_name}_bismark_bt2_PE_report.txt"
+
+hydroxy_R1_bismark_bam="${hydroxy_R1_file_path}_bismark_bt2_pe.bam"
+
+#unmapped_unmapped_R1="${temp_path}/$unmethyl_control/$hydroxymethyl_control/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz"
+#unmapped_unmapped_R2="${temp_path}/$unmethyl_control/$hydroxymethyl_control/${trimmed_R2_file_name}.fastq.gz_unmapped_reads_2.fq.gz_unmapped_reads_2.fq.gz"
+methyl_R1="${temp_path}/$unmethyl_control/$hydroxymethyl_control/$methyl_control/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz"
+methyl_R2="${temp_path}/$unmethyl_control/$hydroxymethyl_control/$methyl_control/${trimmed_R2_file_name}.fastq.gz_unmapped_reads_2.fq.gz_unmapped_reads_2.fq.gz_unmapped_reads_2.fq.gz"
+
+#trimmed_unmapped_R1_bam="${temp_path}/$unmethyl_control/$hydroxymethyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam"
+trimmed_genome_R1_bam="${temp_path}/$unmethyl_control/$hydroxymethyl_control/$methyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam"
 
-trimmed_unmapped_R1_bam="${temp_path}/$unmethyl_control/$hydroxymethyl_control/genome_alignment/${trimmed_R1_file_name}.fastq.gz_unmapped_reads_1.fq.gz_unmapped_reads_1_bismark_bt2_pe.bam"
 echo "temp_path is $temp_path"
-####################previous map_to_control_seqs.sh###########################
+
+#####################previously trim.sh###########################
 ###############################################################################
+if [ ! -f "${temp_path}/$R1_trimmed" ]; then
+    cd $temp_path
+        echo "$R1_trimmed does not exist yet"
+        cutadapt -q 20 -m 15 -a AGATCGGAAGAGC -A AAATCAAAAAAAC -o $R1_trimmed -p $R2_trimmed $R1 $R2 --cores=${SLURM_CPUS_PER_TASK}
```

Comment:
might be better just to use "--cores=0" to auto detect how many cores are available to use?